lib/dotcom_web/components/trip_planner/itinerary_group.ex

```diff
@@ -33,20 +54,59 @@ defmodule DotcomWeb.Components.TripPlanner.ItineraryGroup do
               realtime_state={leg.realtime_state}
             />
           </div>
-          <%= if Enum.count(rest) > 0, do: "Similar trips depart at:" %>
-          <span :for={alternative <- rest}>
-            <%= format_datetime_short(alternative.departure) %>
-          </span>
         </:content>
       </.accordion>
     </div>
     """
   end
 
+  attr :class, :string, default: ""
+  attr :mode, :any, required: true
+  attr :leg, :any, required: true
+
+  defp leg_icon(%{mode: %PersonalDetail{}, leg: %{duration: duration}} = assigns) do
```

Comment:
A few thoughts on this `<.leg_icon />` component:

- I'm not convinced that this is the right place for this, but since this is the only consumer of this component for now, that's where this landed.
- I'm partly thinking that this does belong in the same file as (or at least close to) _some part_ of the blob mentioned above, which is part of why I haven't moved it yet.
- I'm not even a little bit attached to the name `.leg_icon`. It's what came to me as I was building this, and couldn't immediately think of a better name. Open to ideas!

Reply:
I think the name's good and the placement makes sense!

Reply:
I did change this slightly but not much. :D I still think it's okay here for now.